content/en/docs/tasks/administer-cluster/memory-manager.md

```diff
@@ -0,0 +1,390 @@
+
+Two policies can be selected:
+
+* `none` (default)
+* `static`
+
+#### none policy {#policy-none}
+
+This is the default policy and does not affect the memory allocation in any way.
+It acts the same as if the Memory Manager is not present at all.
+
+The `none` policy returns default topology hint. This special hint denotes that Hint Provider (Memory Manger in this case) has no preference for NUMA affinity with any resource.
+
+#### static policy {#policy-static}
+
+In the case of the `Guaranteed` pod, the `static` Memory Manger policy returns topology hints relating to the set of NUMA nodes where the memory can be guaranteed, and reserves the memory through updating the internal [NodeMap][2] object.
+
+In the case of the `BestEffort` or `Burstable` pod, the `static` Memory Manager policy sends back the default topology hint as there is no request for the guaranteed memory, and does not reserve the memory in the internal [NodeMap][2] object.
+
+### Reserved memory flag
+
+[Node Allocatable Feature](https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/) is commonly used by node administrators to reserve K8S node system resources for the kubelet or operating system processes in order to enhance the node stability. A dedicated set of flags can be used for this purpose to set the total amount of reserved memory for a node. This pre-configured value is subsequently utilized to calculate the real amount of node's "allocatable" memory available to pods. Also, K8S scheduler incorporates "allocatable" to optimise pod scheduling process. The foregoing flags include `--kube-reserved`, `--system-reserved` and `--eviction-threshold`. The sum of their values will account for the total amount of reserved memory.    
+
+A new `--reserved-memory` flag was added to Memory Manager to allow for this total reserved memory to be split (by a node administrator) and accordingly reserved across many NUMA nodes. 
+
+The flag specifies a comma-separated list of memory reservations per NUMA node.
+This parameter is only useful in the context of the Memory Manager feature. 
+The Memory Manager will not use this reserved memory for the allocation of container workloads.
+
+For example, if you have a NUMA0 with `10Gi` of memory available, and the `--reserved-memory` was specified to reserve `1Gi` of memory at NUMA0, the Memory Manager assumes that only `9Gi` is available for containers.
+
+You can omit this parameter, however, you should be aware that the amount of reserved memory from all NUMA nodes should be equal to the amount of memory specified by the [Node Allocatable features]((https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/)). If at least one node allocatable parameter is non-zero, you will need to specify `--reserved-memory` for at least one NUMA node. In fact, `eviction-hard` threshold value is equal to `100Mi` by default, so if `static` policy is used, `--reserved-memory` is obligatory.

```

Comment:
nit: use site-relative hyperlinks
```suggestion
You can omit this parameter, however, you should be aware that the amount of reserved memory from all NUMA nodes should be equal to the amount of memory specified by the [Node Allocatable features]((/docs/tasks/administer-cluster/reserve-compute-resources/)). If at least one node allocatable parameter is non-zero, you will need to specify `--reserved-memory` for at least one NUMA node. In fact, `eviction-hard` threshold value is equal to `100Mi` by default, so if `static` policy is used, `--reserved-memory` is obligatory.
```

I would actually link to https://kubernetes.io/docs/concepts/architecture/nodes/#capacity though.

Reply:
Thanks, indeed  https://kubernetes.io/docs/concepts/architecture/nodes/#capacity indirectly points to Node Allocatable features, but "Node Allocatable features" is key as MM directly rely on the parameters of this feature.